### PR TITLE
[Optimize CWV] Add note on lazy-loading and LCP

### PR DESCRIPTION
**Pull request by addyosmani**
Changes proposed in this pull request:

- Adds paragraph + Lighthouse audit reference on not lazy-loading the LCP image (links to article)
- Adds paragraph + Lighthouse audit reference on deferring offscreen images

### CODE REVIEW

**Comment by addyosmani** [comment]
> Do not use master. Affected lines: 210

Note: This is flagged on usage of the URL "https://support.google.com/webmasters/answer/9205520?hl=en", which is intentional.

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/optimize-vitals-lighthouse/index.md [hunk 1]
```diff
@@ -162,6 +161,20 @@ background? Yes.
 Any image flagged as the LCP image whether via CSS background or `<img>` is a candidate if it's
 discovered at a waterfall depth of three or more.
 
+### Lazy-loading offscreen images and avoiding this for LCP
+
+Offscreen images that are not critical to the initial user experience can be [lazy-loaded](/browser-level-image-lazy-loading/). This is a technique that defers downloading an image until a user scrolls near it, which can reduce network contention for critical assets and in some cases improve LCP. The "Defer offscreen images" audit can help here:
```
Comment (malchata) [inline src/site/content/en/blog/optimize-vitals-lighthouse/index.md:166]:
```suggestion
Offscreen images that are not critical to the initial user experience can be [lazy-loaded](/browser-level-image-lazy-loading/). This is a technique that defers downloading an image until a user scrolls near it, which can reduce network contention for critical assets and in some cases improve LCP. The ["Defer offscreen images"](/offscreen-images/) audit can help here:
```

**Review by malchata — APPROVED** [review]
Minor suggestion, but otherwise LGTM!

**Comment by addyosmani** [comment]
Thank you for the review, @mihajlija and @malchata!

**Pull request merged**